scripts/policy/frameworks/storage/backend/redis/main.zeek

```diff
@@ -22,6 +22,15 @@ export {
 		# same server. Defaults to an empty string, but preferably should be set
 		# to a unique value per Redis backend opened.
 		key_prefix: string &default="";
+
+		## Timeout for connection attempts to the backend. Connection attempts
+		## that exceed this time will return
+		## :zeek:see:`Storage::CONNECTION_FAILED`.
+		connect_timeout: interval &default=5 sec;
```

Comment:
```suggestion
		connect_timeout: interval &default=default_connect_timeout;
```

Would make sense to add a module wide default (as const &redef). That way the timeout for every Redis connection that doesn't override it explicitly be changed at once. Same for the operation_timeout.